### PR TITLE
adding github release image

### PR DESCRIPTION
**Pull request by Joseph-Irving**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ pipeline:
 
   publish:
     github_release:
+      image: plugins/github-release
       api_key: FFcsLKh2G/UDlZlcREUmfy94ghWS5Qpja+wq2SDFhRd2CKO4pC13BvRj2n0FmC3CVhWVUC6tfNH43FHmDMP1KzKFGSjdmihbuQgIcAB0cmpDaEmLoeJj1vJxe/OPnQBTQlVNc+fz1vqXeOPPPQz+dyjJ5uApgDFFeRCDeMByUO7PmilZv67oyCnMXMnQmRetptA/kJDz0Hozzv+bH443zSzqPtleTM7nO57gtubhdJr+h/+B1uIPU2KLiLMqimDaA6s50pGRhjGp3/f+Fp8B9q6vEuLVWkD9jffAYoiB9q9m3lY7fsxqKGB1Oe21Bv3jdfREsvsSSm7Dzl/XZA1asT0Si3keNWhlRdI2QQRsenoBIKqe7Zc+wOH4pZsC3xT0iWLbVRJBy1bpPQUDAr4JImARuyV/ZZYZh2358LcM2Ycv68S4a2sZ4AyDSZ0Po2GH1LszcfbXskMw9qZ7Z54WVlC3KOkPXLgGv57HpgIe0I5UzmCmEsVBUjy/t83FRpjJYmsCv+VkG4UtDh6nwxuABR74hm7/at+ckSMPru+8MinKLSksNkBoRPm0pmYptq6EhLfViKEertQ2NJBanAZOQDo2Lwyow9kjs+Xw1Emmf984dlUTFPrizNPTjBHvD7oeEmjnuCgc6MIzh1wWCCq5uEvr3tUh2bW3IZTxAkDccsw=
```
Comment (vaijab) [inline .drone.yml]: What key is that?

Comment (marcinc) [reply .drone.yml]: I believe this is the same key as the one used in travis previously. Potentially need changing?

Comment (vaijab) [reply .drone.yml]: It does not look the same. In travis, the API key string was encrypted. @Joseph-Irving ?

Comment (Joseph-Irving) [reply .drone.yml]: I just copied that api key from .travis.yml

Comment (vaijab) [reply .drone.yml]: Won't work :-)

Comment (Joseph-Irving) [reply .drone.yml]: ah ok, I'll generate a new one. I'll Store it as a secret as well instead of being in the .drone.yml

**Comment by Joseph-Irving** [comment]
I have removed the api key from the .drone.yml and added a new one as a drone secret instead

**Comment by vaijab** [comment]
@Joseph-Irving do GH releases work now?

**Comment by Joseph-Irving** [comment]
it sure does, I have tested it now, took me a little while to get it properly working

**Review by vaijab — APPROVED** [review]

**Pull request merged**